unittests/test_policies.py

```diff
@@ -109,26 +109,23 @@ def num_failures_stage(runner, stage):
 
 def open_json_schema():
```

Comment:
I don't see the purpose of this function. I would merge it with the `validate_report()`.